lib/sensors/temperature.cpp

```diff
@@ -0,0 +1,86 @@
+#include "temperature.hpp"
+
+namespace hyped::sensors {
+
+Temperature::Temperature(hyped::core::ILogger &log, io::I2c &i2c, const std::uint8_t channel)
+    : logger_(log),
```

Comment:
Naming convention for logger is now `log_`, needs changed

Reply:
`logger_` I believe